packages/lodestar-cli/src/cmds/account/cmds/wallet/utils.ts

```diff
@@ -0,0 +1,46 @@
+import fs from "fs";
+import path from "path";
+import {IGlobalArgs} from "../../../../options";
+import {YargsError, writeFile600Perm, randomPassword, readPassphraseFile, initBLS} from "../../../../util";
+import {WalletManager} from "../../../../wallet";
+import {getAccountPaths} from "../../paths";
+import {IWalletRecoverArgs} from "./recover";
+
+export async function createWalletFromArgsAndMnemonic(
```

Comment:
If `packages/lodestar-cli/src/cmds/account/cmds/wallet/recover.ts` is the only one using this logic it's not quite useful to split the logic in this utility. I would just dump this logic in recover, and if in the future it is used in multiple places, then we can abstract it away.